man/__init__.py

```diff
@@ -20,7 +20,11 @@
 
 def get_parser():
     """Return OptionParser instance used by manpage generator."""
-    parser = toolopts.CLI()._parser
+    cli = toolopts.CLI()
+    # Show the subcommands in the SYNOPSIS section, but hide in the
+    # `convert2rhel convert` section.
+    cli._convert_parser.usage = cli.usage(include_subcommands=False)
+    parser = cli._parser
```

Comment:
```suggestion
    parser = toolopts.CLI()._parser
```
Let's show the `convert` subcommand everywhere and mention in the help string for the subcommand that if no subcommand is specified, `convert` is used as a default.

Reply:
updated!